From 7c17fa8fccfbca5096338c29efe182b6a6d618dc Mon Sep 17 00:00:00 2001
From: Mervin Fansler <mmfansler@gmail.com>
Date: Tue, 30 May 2023 22:36:05 -0400
Subject: [PATCH] declare cxx14

---
 src/Makevars.win | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/Makevars.win b/src/Makevars.win
index 353270f..48a70f5 100644
--- a/src/Makevars.win
+++ b/src/Makevars.win
@@ -1,6 +1,7 @@
 PKG_CPPFLAGS = -DRCPP_USE_UNWIND_PROTECT -DRCPP_NO_RTTI -DPCRE2_CODE_UNIT_WIDTH=8 -DHAVE_CONFIG_H -DPCRE2_BUNDLED -I. -IPCRE2
 PKG_CXXFLAGS = -DRCPP_PARALLEL_USE_TBB=1
 PKG_LIBS     = -lpthread -L. -lSFPCRE2 $(shell "${R_HOME}/bin${R_ARCH_BIN}/Rscript.exe" -e "RcppParallel::RcppParallelLibs()")
+CXX_STD = CXX14
 
 LIBPCRE2 = PCRE2/pcre2_chartables.o \
 	PCRE2/pcre2_auto_possess.o \
